src/Data/Functor/Wrap.purs

```diff
@@ -0,0 +1,27 @@
+module Data.Functor.Wrap where
+
+import Prelude
+
+import Data.Functor.FunctorRight (class FunctorRight, rmap)
+import Data.Newtype (class Newtype)
+
+-- | Given a type that takes two type parameters, `p`, provides a `Functor`
```

Comment:
From my reading this isn't accurate; a Functor instance can only ever map over the last type parameter.

Reply:
The real function of Wrap here is to provide a Functor instance for a data type with 2 type parameters given only a FunctorRight instance. But I can't think of a good reason why someone would provide only FunctorRight and not Functor, so I'm actually wondering if now isn't a good time to just drop Wrap entirely?

Reply:
Yeah, dropping it sounds good. If we ever needed to readd it, that's not a breaking change.